src/chain_spec.rs

```diff
@@ -263,6 +263,11 @@ impl ChainSpec {
         }
     }
 
+    /// Returns the number of bytes that the "block number" field of various data structures uses.
```

Comment:
```suggestion
    /// Returns the number of bytes that the "block number" field of various data structures uses. If the field is missing, the value defaults to 4.
```

Reply:
I've intentionally omitted this, because this API is more about obtaining the information that can be computed from the chain spec, rather than getting the content of the chain spec.
If it was the latter, then you'd need to be able to differentiate "4 by default" and "4 as explicitly provided" for example.